lib/membrane_rtc_engine/track.ex

```diff
@@ -9,8 +13,18 @@ defmodule Membrane.RTC.Engine.Track do
 
   @type id :: String.t()
   @type encoding :: :OPUS | :H264 | :VP8
-  @type format :: :RTP | :raw | :RTMP
+  @type format :: atom()
 
+  @typedoc """
+  This module contains:
+  type - track can be of type audio or video,
+  stream_id - track can belong to some media stream. Each media stream can contains multiple tracks. This allow to group tracks
+  id - allow to identify track
+  encoding -  describe in what codec media is encodec e.g. OPUS, VP8
+  format - describe in what format media is sent e.g. raw, RTP, RTMP
+  fmtp - struct describing format specific parameters e.g. for H264 it contains profile_level_id
+  disabled? - describe if track is still enabld (peer is still in room and sending media)
```

Comment:
I would suggest following explanation
```suggestion
  This module contains:
  * `type` - audio or video,
  * `stream_id` - media stream this track belongs to. Relationship between tracks (e.g. audio and video)
  can be indicated by assigning each of them the same `stream_id`. One `stream_id` can be assign to any
  number of tracks.
  * `id` - track id
  * `encoding` - track encoding
  * `format` - list of available track formats. At this moment max two formats can be specified.
  One of them has to be `:raw` which indicates that other Endpoints will receive this track in format
  of `encoding`. The other one can be any atom (e.g. `:RTP`).
  * `fmtp` - struct describing format specific parameters e.g. for H264 it contains `profile_level_id`
  * `active?` - indicates whether track is still available or not (because peer left a room)
  ```